src/styles/main.scss

```diff
@@ -0,0 +1,4 @@
+@import url('https://fonts.googleapis.com/css?family=Advent+Pro:100,200,300,400,500,600,700');
+
+@import "Reset";
+@import "Header"; 
```

Comment:
looks great! thanks for setting up the scss. i hate trying to get the module to work.